ldpd/lde.c

```diff
@@ -1708,6 +1721,38 @@ on_get_label_chunk_response(uint32_t start, uint32_t end)
 		current_label_chunk = listtail(label_chunk_list);
 }
 
+void
+lde_free_label(uint32_t label)
+{
+	struct listnode	*node;
+	struct label_chunk	*label_chunk;
+	uint64_t		pos;
+
+	node = listhead(label_chunk_list);
+
+	while (node) {
+		label_chunk = listgetdata(node);
+
+		if (label_chunk != NULL)
```

Comment:
Couldn't we use a for loop instead? Maximum line length for ldpd is 80 chars.

Example: `for (node = listhead(label_chunk_list); node; node = listnextnode(node))`.

Better yet, you can use the `ALL_LIST_ELEMENTS_RO` macro.